supplier_api_tester/supplier_api_tester/cli.py

```diff
@@ -36,59 +32,20 @@ def supplier_tester(url, api_key, product_id, timeslots, availability, reservati
     if any((availability, timeslots, reservation, booking)) and not product_id:
         product_id = click.prompt('Product ID')
 
-    if availability:
-        print_title('AVAILABILITY TESTS')
-        runner = SupplierApiTester(
-            host=url,
-            api_key=api_key,
-            product_id=product_id,
-            test_target='availability',
-        )
-        results = runner.run()
-        results_printer(results, no_colors)
-
-    if reservation:
-        print_title('RESERVATION TESTS')
-        runner = SupplierApiTester(
-            host=url,
-            api_key=api_key,
-            product_id=product_id,
-            test_target='reservation',
-        )
-        results = runner.run()
-        results_printer(results, no_colors)
-
-    if booking:
-        print_title('BOOKING TESTS')
-        runner = SupplierApiTester(
-            host=url,
-            api_key=api_key,
-            product_id=product_id,
-            test_target='booking',
-        )
-        results = runner.run()
-        results_printer(results, no_colors)
-
-    if catalog:
-        print_title('PRODUCT CATALOG')
-        runner = SupplierApiTester(
-            host=url,
-            api_key=api_key,
-            product_id=product_id,
-            test_target='catalog',
-        )
-        results = runner.run()
-        results_printer(results, no_colors)
+    if version_1:
+        supplier_tester_v1(url, api_key, product_id, timeslots, availability, reservation, booking, catalog, no_colors)
+    else:
+        supplier_tester_v2(url, api_key, product_id, availability, reservation, booking, catalog, no_colors)
 
 
 @click.command()
 @click.option('-u', '--url', required=True, prompt='Server URL', type=str)
 @click.option('-k', '--api-key', required=True, prompt='API Key', type=str)
-def supplier_products(url, api_key):
+@click.option('-v1', '--version-1', is_flag=True, default=False, help='Run the CLI testing tools for API v1.x')
```

Comment:
Instead of flag I would use an integer
```py
@click.option('-v', '--version', default=2, help='Choose the supplier API version')
```
So at some point, we could use `-v 3`